tembo-pgmq-python/Makefile

```diff
@@ -16,4 +16,4 @@ run-pgmq-postgres:
 	docker run -d --name pgmq-postgres -e POSTGRES_PASSWORD=postgres -p 5432:5432 quay.io/tembo/pgmq-pg:latest || true
 
 test: run-pgmq-postgres
-	poetry run pytest
+	poetry run python -m unittest discover tests
```

Comment:
`pytest` vs `unittest` is certainly a debate. Curious to hear your thoughts on why `unittest` over `pytest` for this project.

Reply:
The main reason I chose `unittest` is because it has fewer dependencies. Since `unittest` is part of the Python standard library, we don't need to install anything extra. Another reason is that with new versions of Python, like 3.13 and above, there are many changes, such as removing the GIL and using JIT (Just In Time) compilation. We should wait for `pytest` to update and support these new features. Additionally, `unittest` has good test discovery built-in, which helps us easily find and run our tests.

Reply:
Thank you. 

> Another reason is that with new versions of Python, like 3.13 and above, there are many changes, such as removing the GIL and using JIT (Just In Time) compilation.

I agree that there are new versions of python being released, but I'm not sure how pytest relates. Does `pytest` not run on python 3.13? 

Overall though, I don't think there is anything wrong with pytest, it is a more complete test framework than unittest.  That being said, I'm ok with moving to `unittest` if that means we get more complete test coverage on the project.

Reply:
If we are going to change the test framework to `unittest` in this PR, then we'll also need to update the [Makefile](https://github.com/tembo-io/pgmq/blob/6f3e1e5af9c81be323550908d2994fc625f191f3/tembo-pgmq-python/Makefile#L18-L19). This is called in the [CI environment](https://github.com/tembo-io/pgmq/blob/6f3e1e5af9c81be323550908d2994fc625f191f3/.github/workflows/pgmq_python.yml#L52-L53)

Reply:
I generally prefer to avoid adding external dependencies. pytest is an external package, while unittest is built into Python and does not require installation. Since we are not using any pytest plugins, I believe it's best to keep things simple and stick with unittest unless pytest becomes absolutely necessary.

Reply:
Thank you. 🤦 my bad!